src/rp2/in_transaction.py

```diff
@@ -206,3 +206,6 @@ def is_crypto_fee_defined(self) -> bool:
 
     def is_taxable(self) -> bool:
         return self.transaction_type.is_earn_type()
+
+    def __lt__(self, other: "InTransaction") -> bool:
```

Comment:
What is this used for?

Reply:
Needed to define a sort order on transactions: https://docs.python.org/3/library/functions.html#sorted

Reply:
Done